### PR TITLE
459 automatic epidata generation for simulations

### LINKED ISSUE #459: data generation for simulations is time-consuming and not user friendly
Before running a simulation in `cpp/simulations`, it is nesseccary to create a lot of data in [memilio-epidata](https://github.com/DLR-SC/memilio/tree/main/pycode/memilio-epidata).
In addition, the user must set the moving average = 7 there during generation.

The work process can be simplified if we provide a script for generation.

### PR DESCRIPTION
**Pull request by HenrZu**
## Merge Request - GuideLine Checklist 

**Guideline** to check code before resolve WIP and approval, respectively.
As many checkboxes as possible should be ticked.

### Checks by code author:
Always to be checked:
* [x] There is at least one issue associated with the pull request.
* [x] The branch follows the naming conventions as defined in the [git workflow](git-workflow).
* [x] New code adheres with the [coding guidelines](coding-guidelines)
* [x] Make sure that the [pre-commit linting/style checks pass](https://github.com/DLR-SC/memilio/wiki).

If functions were changed or functionality was added:
* [ ] Tests for new functionality has been added
* [ ] A local test was succesful

If new functionality was added:
* [ ] There is appropriate **documentation** of your work. (use doxygen style comments)

If new third party software is used:
* [ ] Did you pay attention to its license? Please remember to add it to the wiki after successful merging.

If new mathematical methods or epidemiological terms are used:
* [ ] Are new methods referenced? Did you provide further documentation? Has the glossary been updated? 

The following questions are addressed in the documentation if need be: 
* [ ] Developers (what did you do?, how can it be maintained?)
* [ ] For users (how to use your work?)
* [ ] For admins (how to install and configure your work?)

* For documentation: Please write or update the Readme in the current working directory!

### Checks by code reviewer(s):
* [ ] Is the code clean of development artifacts e.g., unnecessary comments, prints, ...
* [ ] The ticket goals for each associated issue are reached or problems are clearly addressed (i.e., a new issue was introduced).
* [ ] There are appropriate **unit tests** and they pass.
* [ ] The git history is clean and linearized for the merge request. All reviewers should squash commits and write a simple and meaningful commit message.
* [ ] Coverage report for new code is acceptable. 

Closes #459

### CODE REVIEW

File: cpp/simulations/data_generation.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+cd ../..
+data_dir=$PWD/data/pydata
+
+# download data
+cd "pycode/memilio-epidata"
+python setup.py install
+python memilio/epidata/getPopulationData.py -o $data_dir
+python memilio/epidata/getDIVIData.py -o $data_dir -m 7
+python memilio/epidata/getSimulationData.py -o $data_dir -m 7
+python memilio/epidata/getCaseData.py -o $data_dir -m 7
```
Comment (mknaranja) [inline cpp/simulations/data_generation.sh:21]: https://github.com/DLR-SC/memilio/blob/main/pycode/memilio-epidata/memilio/epidata/getSimulationData.py already calls the other functions. Why did you call them explicitly?

File: cpp/simulations/data_generation.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
+# Run all scripts in memilio-epidata which are necessary to run the simulation files.
+
+# activate virutal_env 
+# The path to the virtual environment has to be adjusted individually
+path_virtual_env="$HOME/Documents/virtualenv/bin/activate"
```
Comment (mknaranja) [inline cpp/simulations/data_generation.sh:6]: This only works on Linux, right?

File: cpp/simulations/data_generation.sh (hunk 1 above)
Comment (HenrZu) [reply cpp/simulations/data_generation.sh]: This was for some tests. Now only getSimulationData.py is called.

File: cpp/simulations/data_generation.sh (hunk 2 above)
Comment (HenrZu) [reply cpp/simulations/data_generation.sh:6]: Only the path has to be adjusted. Otherwise, the script runs without an error on windows and linux.

**Pull request closed without merging**